### PR TITLE
JDL changes.

### PR DESCRIPTION
**Pull request by mshima**
Contains the following changes:
- When importing entities only jdls, read `.yo-rc.json` on demand, current it requires `import-jdl` command to always pass `.yo-rc.json` data.
- Stop creating creationTimestamp
- When creationTimestamp exists in jdl, it should be passed as an int.
- Stop creating changelogDate.

---

Please make sure the below checklist is followed for Pull Requests.

-   [ ] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
-   [ ] Tests are added where necessary
-   [ ] [jhipster-online](https://github.com/jhipster/jhipster-online) was updated if necessary
-   [ ] Documentation is added/updated where necessary
-   [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/main/CONTRIBUTING.md) are followed

When you are still working on the PR, consider converting it to Draft (bellow reviewers) and adding `skip-ci` label, you can still see CI build result at your branch.

### CODE REVIEW

File: jdl/jdl-importer.js [hunk 1]
```diff
@@ -183,28 +166,30 @@ function checkForErrors(jdlObject, configuration) {
...
-        databaseType = configuration.application['generator-jhipster'].databaseType;
-        creationTimestamp = configuration.application['generator-jhipster'].creationTimestamp;
+    let application = configuration.application;
+    if (!configuration.application && doesFileExist('.yo-rc.json')) {
+        application = readJSONFile('.yo-rc.json');
+    }
+    if (application && application['generator-jhipster']) {
+        if (applicationType === undefined) applicationType = application['generator-jhipster'].applicationType;
+        if (applicationName === undefined) applicationName = application['generator-jhipster'].baseName;
+        if (databaseType === undefined) databaseType = application['generator-jhipster'].databaseType;
```
Comment (MathieuAA) [inline jdl/jdl-importer.js:178]: Please use brackets

File: jdl/utils/format-utils.js [hunk 2]
```diff
@@ -48,57 +45,3 @@ function formatComment(comment) {
...
 }
-
-/**
- * Formats an optional date to be used by Liquibase.
- * @param {Object} args - the function's arguments.
- * @param {Date} args.date - the date to format, optional.
- * @param {Number} args.increment - an increment to be used to set minutes, optional.
- * @return {string} the formatted Date.
- */
-function formatDateForLiquibase(args) {
```
Comment (MathieuAA) [inline jdl/utils/format-utils.js:59]: awesome :)

**Review by MathieuAA — CHANGES_REQUESTED** [review]

File: jdl/jdl-importer.js (hunk 1 above)
Comment (mshima) [reply jdl/jdl-importer.js]: @MathieuAA I suppose you are asking for curly brackets.
Done

**Review by MathieuAA — APPROVED** [review]

**Comment by mshima** [comment]
@MathieuAA thanks.

**Pull request merged**